sdk/src/sharedTest/java/io/customer/sdk/queue/QueueStorageIntegrationTest.kt

```diff
@@ -109,6 +109,19 @@ class QueueStorageIntegrationTest : BaseTest() {
         queueStorage.getInventory().count() shouldBeEqualTo 0
     }
 
+    @Test
+    fun givenDeleteGroupTask_expectUpdateInventory() {
+        val givenStartOfTheGroup = QueueTaskGroup.IdentifyProfile(String.random)
+
+        queueStorage.create(String.random, String.random, givenStartOfTheGroup, null)
+        queueStorage.create(String.random, String.random, null, listOf(givenStartOfTheGroup))
+        queueStorage.create(String.random, String.random, null, listOf(givenStartOfTheGroup))
+
+        val itemsDeleted = queueStorage.deleteGroup(givenStartOfTheGroup.toString())
+        itemsDeleted.count() shouldBeEqualTo 3
+        queueStorage.getInventory().count() shouldBeEqualTo 0
```

Comment:
Shouldn't we also need to add test for case when there are other tasks in the queue as well to make sure only the desired tasks have been deleted?

Reply:
I added those tests, but forgot to push them 😆 
![Screenshot 2023-03-02 at 9 31 32 PM](https://user-images.githubusercontent.com/4658866/222491846-3fa0c2a2-1f38-49be-9162-de99c83dd3a6.png)